docs/admin/system-information.rst

```diff
@@ -1777,6 +1777,9 @@ How to reindex
     cr> INSERT INTO rx.tmp_metrics (id, temperature) (SELECT id, temperature FROM rx.metrics);
     INSERT OK, 2 rows affected (... sec)
 
+    cr> REFRESH TABLE rx.tmp_metrics;
```

Comment:
The change here is a bit misleading. Given that we no longer re-allocate the shards, we also don't need to wait for allocation anymore. This in turn has the effect that the swap statement can return before the metadata update is propagated through the cluster and a query after the swap can operate on the old metadata.

We should probably revert this and change the swap action to wait for state propagation somehow.